### DISCUSSION TITLE
Unable to Get GitHub Education Benefits after Verification | 8 days over

### DISCUSSION THREAD

**Discussion by tejkiran-1** [community/community#136286] [GitHub Education]
### Hi everyone,

Hello GitHub Community, I recently verified my student status through GitHub Education, but it has been eight days since my verification, and I have yet to receive the benefits. According to GitHub Education's policy, I should receive these benefits within three days, with a maximum delay of five days. Given that this timeframe has already been exceeded, I am reaching out here to seek assistance or advice on how to resolve this issue. Any guidance or support would be greatly appreciated. Thank you.

**Comment by davevad93** [comment]
Hi @tejkiran-1 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @tejkiran-1 was approved and your benefits are now available.`

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
